resources/conanfile.py

```diff
@@ -0,0 +1,65 @@
+import os
+
+from conan import ConanFile
+from conan.tools.files import copy, update_conandata
+from conan.tools.scm import Version
+from conan.errors import ConanInvalidConfiguration
+
+required_conan_version = ">=1.58.0 <2.0.0"
+
+
+class CuraResource(ConanFile):
+    name = "cura_resources"
+    license = ""
+    author = "UltiMaker"
+    url = "https://github.com/Ultimaker/cura-private-data"
+    description = "Cura Resources"
+    topics = ("conan", "cura")
+    exports = "LICENSE*"
```

Comment:
the license is located one directory up, is that an issue? Also this conan package is just data, as I understand it data is not copy-rightable. Do we even need a license?

Reply:
can't hurt to have the license here, will change